src/Components/IndividualReview.jsx

```diff
@@ -10,26 +10,65 @@ const IndividualReview = () => {
   const { ID } = useParams();
   const [review, setReview] = useState({});
   const [comments, setComments] = useState([]);
-
-  console.log(ID, "<<ID");
+  const [votes, setVotes] = useState(0);
+  const [err, setErr] = useState(null);
 
   useEffect(() => {
     axios
       .get(`https://josies-games.herokuapp.com/api/reviews/${ID}`)
       .then((res) => {
         setReview(res.data.review);
+        setVotes(res.data.review.votes);
       });
-  }, [ID]);
+  }, [ID, votes]);
+
+  console.log(votes, "<<<votes");
 
   useEffect(() => {
     axios
       .get(`https://josies-games.herokuapp.com/api/reviews/${ID}/comments`)
       .then((res) => {
-        console.log(res.data.comments, "<<<<res.data.comments");
         setComments(res.data.comments);
       });
   }, [review]);
 
+  function downVoteClick() {
+    setVotes((current) => current - 1);
+    axios
+      .patch(`https://josies-games.herokuapp.com/api/reviews/${ID}`, {
+        inc_votes: -1,
+      })
+      .then(() => {
+        console.log("fulfilled");
+        setErr(null);
+      })
+      .catch((err) => {
+        console.log(err);
+        setVotes((current) => current + 1);
+        setErr("Sorry, something went wrong. Please try again.");
+      });
+  }
+
+  function upVoteClick() {
+    setVotes((current) => current + 1);
+    axios
+      .patch(`https://josies-games.herokuapp.com/api/reviews/${ID}`, {
+        inc_votes: 1,
```

Comment:
`downVoteClick` and `upVoteClick` look very similar - could you make this more DRY?

you can pass the differences (`+1` or `-1`) through as arguments